server/aws-lsp-codewhisperer/src/language-server/inline-completion/codeWhispererServer.ts

```diff
@@ -594,6 +594,22 @@ export const CodewhispererServerFactory =
 
             sessionManager.closeSession(session)
 
+            let translatedError = error
+
+            if (hasConnectionExpired(error)) {
+                translatedError = new AmazonQServiceConnectionExpiredError(getErrorMessage(error))
+            }
+
+            if (translatedError instanceof AmazonQError) {
+                throw new ResponseError(
+                    LSPErrorCodes.RequestFailed,
+                    error.message || 'Error processing suggestion requests',
+                    {
+                        awsErrorCode: (error as AmazonQError).code,
+                    }
```

Comment:
`message` and `code` are pulled from `error`. 
Should these be pulled from `translatedError`?